builtin/credential/approle/path_role.go

```diff
@@ -1377,6 +1522,54 @@ func (b *backend) pathRoleBoundCIDRListDelete(ctx context.Context, req *logical.
 	return nil, b.setRoleEntry(ctx, req.Storage, roleName, role, "")
 }
 
+func (b *backend) pathRoleSecretIDBoundCIDRDelete(ctx context.Context, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
```

Comment:
This function and the next are nearly identical. Maybe you can extract the common core into a parameterized version.

Reply:
Hah, for some reason yesterday I didn't think so but looking at it this morning it's pretty obvious it'd be easy to do. :-) Will update.